test/test-event-dom.js

```diff
@@ -15,7 +15,7 @@ function countListeners(target, type) {
   return listeners.filter(listener => listener.type == type).length;
 }
 
-exports['test window close clears listeners'] = function(assert) {
+exports['test window close clears listeners'] = function*(assert) {
```

Comment:
maybe I should've commented here, sorry for long delay.
Those are required because of the change in Part 2.
Those functions are passed to isGenerator function, so all legacy generator in tests should be replaced with ES6 generator.